R/EXADBI-internal.R

```diff
@@ -8,10 +8,11 @@
 
 
 
-# Tries to extrapolate the schema name from the statement given. It looks for strings framed by 'from' and a dot (not case sensitive).
-#
-# @param stmt The statement used
-# @return a character vector containing all schemas found, or alt
+#' Tries to extrapolate the schema name from the statement given. It looks for strings framed by 'from' and a dot (not case sensitive).
+#'
+#' @param stmt The statement used
+#' @return a character vector containing all schemas found, or alt
```

Comment:
```suggestion
#' @return A character vector containing all schemas found, or alt
```

Reply:
This sentence seems to be incomplete, `or alt`. I think it is typo for `halt` but I was not sure.